Std/CodeAction/Deprecated.lean

```diff
@@ -0,0 +1,69 @@
+/-
+Copyright (c) 2023 Mario Carneiro. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Mario Carneiro
+-/
+import Lean.Server.CodeActions
+import Std.CodeAction.Basic
+import Std.Lean.Position
+
+/-!
+# Code action for @[deprecated] replacements
+
+This is an opt-in mechanism for making machine-applicable `@[deprecated]` definitions. When enabled
+(by setting the `machineApplicableDeprecated` tag attribute), a code action will be triggered
```

Comment:
Could you say what command the user is actually meant to type?

Reply:
As far as this file is concerned, it makes no user-visible syntax. You just call `machineApplicableDeprecated.tag` directly (and this is public API, available for other tactic writers). If one includes the `Alias.lean` file, the user syntax is `@[deprecated] alias foo := bar`.